google-tests/soltrace_example_tests/st_hfsf_sim_run_test.cpp

```diff
@@ -0,0 +1,107 @@
+#include <gtest/gtest.h>
+
+#include "strace.h"
+#include "stapi.h"
+#include "split_csv.h"
+
+using namespace std;
+
+TEST(st_hfsf_sim_run_opt_test, BasicAssertions)
```

Comment:
Can you remove all underscores from all test suite names and test names?
See gtest FAQ: https://google.github.io/googletest/faq.html

```suggestion
TEST(HighFluxSolarFurnace, RaysTraced1e5OpticalErrorsOff)
```